doc/source/conf.py

```diff
@@ -167,11 +167,12 @@
     "bootstrap_version": "3",
     "nosidebar": True,
     "navbar_links": [
-        ("Gallery", "auto_examples/index"),
         ("Installation", "installation"),
-        ("About us", "aboutus"),
-        ("Versions", "releases"),
-        ("Code doc", "tofu"),
+        ("Contributing", 'contributing'),
+        ("Gallery", "auto_examples/index"),
+        ("About", "aboutus"),
+        ("Releases", "releases"),
+        ("API", "tofu"),
```

Comment:
The final version of this should be (*partially*) based on big projects. I think this is the direction most open source projects aiming, to have a homogeneity across all projects.